Pipfile

```diff
@@ -6,8 +6,9 @@ name = "pypi"
 [packages]
 flask = "*"
 gunicorn = "*"
-SPARQLWrapper = "*"
 jsonschema = "*"
+sparqlwrapper = "*"
```

Comment:
Can you keep with SPARQLWrapper please. I know it's odd but we had differing behaviour of some kind between "sparqlwrapper" and "SPARQLWrapper" on the other branch.

Could have been something else going on (they seem the same on the surface) but if we change to the lower case install it'd be better to do it after it all gets merged together (better safe than sorry).